File: java/arquivos/LerConteudoArquivo.java
/**
 * Programa Java usa o componente Arquivo para ler o conteúdo do arquivo teste.txt
 * O arquivo manipulado por esse programa fica contido no diretório corrente.
 *
 * Objetivos:
 *   - ler o conteúdo no arquivo da pasta corrente.
 *
 * Módulo: Lógica de programação | Capítulo: XXX | Página: XXX
 * Author: Eder Magalhães
 */
public class LerConteudoArquivo {

    public static void main(String[] args) {
        String nomeArquivo = "teste.txt";
        Arquivo meuArquivo = Arquivo.abrirLeitura(nomeArquivo);

        System.out.println("Início do conteúdo do arquivo " + nomeArquivo + ".\n");
        System.out.println("--------------------------------------");

        String linha = Arquivo.lerLinha(meuArquivo);
        //enquanto existir conteúdo, imprima
        while (linha != null) {
            System.out.println(linha);
            linha = Arquivo.lerLinha(meuArquivo);
        }

        Arquivo.fechar(meuArquivo);
        System.out.println("--------------------------------------");
        System.out.println("\nFim do conteúdo do arquivo " + nomeArquivo + ".");
    }

}

File: python/arquivos/lerConteudoArquivo.py
# -*- coding: utf-8 -*-

# Programa Python demonstra como ler o conteúdo do arquivo teste.txt.
# Objetivos:
#   - ler o conteúdo no arquivo da pasta corrente.
#
# Módulo: Lógica de programação
# Author: Eder Magalhães

nomeArquivo = "teste.txt"

meuArquivo = open(nomeArquivo, 'r');

print "Início do conteúdo do arquivo ", nomeArquivo,"\n"

print '-'*45

for line in meuArquivo:
    print line.strip();

print '-'*45
print "\nFim do conteúdo do arquivo ", nomeArquivo

meuArquivo.close();

